ops/ccs-ops-misc/alert-audit/requirements.txt

```diff
@@ -0,0 +1,2 @@
+pydantic
```

Comment:
`pydantic` eh? Pretty common, but novel to BFD! If this solution is otherwise acceptable, we owe @StewGoin the courtesy of letting him know 🙂

Reply:
@StewGoin OK to use this library? https://docs.pydantic.dev/latest/